base-module.gradle

```diff
@@ -48,6 +48,8 @@ android {
 dependencies {
     coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.1.5'
 
+    implementation Core.splashScreen
```

Comment:
I believe u don't need this in here, check it out and confirm please🙏

Reply:
Yes, you're right.